modules/meta/arrow-meta/src/main/java/arrow/common/utils/AbstractProcessor.kt

```diff
@@ -50,9 +50,15 @@ abstract class AbstractProcessor : KotlinAbstractProcessor(), ProcessorUtils, Ko
       @Suppress("SwallowedException")
       try {
         val path = kDocLocation.toPath()
-        Files.createDirectories(path.parent)
-        Files.delete(path)
-        Files.createFile(path)
+        @Suppress("SwallowedException")
+        try { Files.createDirectories(path.parent) } catch (e : IOException) {
+        }
+        @Suppress("SwallowedException")
+        try { Files.delete(path) } catch (e : IOException) {
+        }
+        @Suppress("SwallowedException")
+        try { Files.createFile(path) } catch (e : IOException) {
+        }
```

Comment:
*squint* what

Reply:
yep, tried with file permissions but it does not work so we have to silently fail if it can't create the file because a previous annotation processor already created it.